src/apps/explorer/styled.ts

```diff
@@ -46,9 +46,14 @@ export const MainWrapper = styled.div`
   ${media.xSmallDown} {
     max-width: 100%;
     flex-grow: 1;
+
     header {
-      margin-left: auto;
-      margin-right: auto;
+      img {
+        max-width: 98%;
+      }
+      div.network-selector-container {
+        padding-right: 2rem;
+      }
```

Comment:
Wouldn't make sense instead to set that property onto the `SelectorContainer` component instead?

Reply:
Yes, I have taken the responsibility for the style to each component.